cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,23 @@
+import {UploadPage}
+  from "../page/index";
+
+describe("upload a file then download another one", () => {
+  let uploadPage: UploadPage;
+
+  before(() => {
+    uploadPage = new UploadPage;
+  });
+
+  it("then the file should be uploaded", () => {
+    // Arrange
+    const file: string = "the-song.txt";
+    const validationMessage: string = "File Uploaded!";
+    uploadPage.visitUploadDemoSite();
+
+    // Action
+    uploadPage.uploadFile(file);
+
+    // Assertion
+    uploadPage.validateUpload(validationMessage);
```

Comment:
instead of validating the title of the page after the upload is done, please validate that the page contains the filename of the file uploaded:
![image](https://user-images.githubusercontent.com/10209180/180503697-f020faa3-5807-44f9-a841-653162c5f584.png)